en/docs/api-gateway/1.1.0/deployment/deployment-modes/kubernetes/kubernetes-standalone.md

```
@@ -261,6 +264,13 @@ spec:
 EOF
 ```
 
+### Expose the data plane
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)